src/applications/disability-benefits/all-claims/pages/individualsInvolvedFollowUp.js

```diff
@@ -51,8 +51,8 @@ export const uiSchema = index => ({
             labels: {
               killedInAction: 'Killed in action',
               woundedInAction: 'Wounded in action',
-              killedInNonBattle: 'Killed non-battle',
-              injuredInNonBattle: 'Injured non-battle',
+              killedNonBattle: 'Killed non-battle',
```

Comment:
would it make sense to map this out in `/constants` since we're using it in multiple places?

Reply:
Are we using it in multiple places?  I think these are just the display values and aren't needed anywhere outside of the UI schema.